docs/development/design/stack.md

```diff
@@ -0,0 +1,60 @@
+# Nexodus Stack
+
+## Summary
+
+This proposal documents the decisions made around the Nexodus software stack.
+
+## Proposal
+
+### Kubernetes Only
+
+#### Background
+
+Early on, we used Docker Compose for orchestration.
+As the stack and complexity grew, we decided to exclusively support Kubernetes, requiring local development to use KIND.
+
+To support use of KIND for local development, and OpenShift for production deployment we settled on using Kustomize.
+
+#### Decisions
+
+- We only support deployment on K8s
+- Kustomize is our deployment configuration tool
+- All manifests in `deploy/base` MUST be useable in any K8s distribution.
+- We allow operators to be used, provided they may also be installed on any K8s distribution.
```

Comment:
How about also:

- While we may use OpenShift for the project's production cluster, we will not make any changes that require the use of OpenShift to run a production instance of the service.